installers/opencv.ps1

```diff
@@ -0,0 +1,17 @@
+wget -O opencv_contrib.zip https://github.com/opencv/opencv_contrib/archive/4.x.zip
+wget -O opencv.zip https://github.com/opencv/opencv/archive/4.x.zip
+Expand-Archive -Path ./opencv.zip
+Expand-Archive -Path ./opencv_contrib.zip
+#downloading opencv and extracting it
+
+New-Item -ItemType Directory -Path ./build
+set-location ./build
+
+$CMAKE_OPTIONS = '-DBUILD_PERF_TESTS:BOOL=OFF -DBUILD_TESTS:BOOL=OFF -DBUILD_DOCS:BOOL=OFF  -DWITH_CUDA:BOOL=OFF -DBUILD_EXAMPLES:BOOL=OFF -DINSTALL_CREATE_DISTRIB=ON'
+$CMAKE_GENERATOR_OPTIONS = '-G "Ninja"' #switch to whatever generator you use but the ones with long names is not working for some stupid reason
+$P_REPO_LOCATION = 'C:\Users\ourpc\CLionProjects\Polarity-' #change ya path to whereever polarity root folder is
```

Comment:
You could just change this to `Get-Location` tbh
Saves the user needing to edit the file and potentially getting the directory wrong